doc/ref_cert/lfn/chapters/chapter01.md

```diff
@@ -4,14 +4,39 @@
 
 ## Table of Contents
-* [1.1 Overview](#1.1)
+* [1.1 Introduction](#1.1)
 * [1.2 Scope](#1.2)
-* [1.3 Resources & References](#1.3)
+* [1.3 Principles and Guidelines](#1.3)
+  * [1.3.1 Overarching Objectives and Goals](#1.3.1)
+* [1.4	Best Practices](#1.4)
+* [1.5 Verification methodologies](#1.5)
+*	[1.6 Assumptions & Dependencies](#1.6)
+*	[1.7 Results Collation & Presentation](#1.7)
+*	[1.8 Governance](#1.8)
+* [x.x Resources & References](#x.x)
+
+## Synopsis
+
+Ensure Reference Implementation of CNTT Reference Model and CNTT Reference Architecture meets industry driven quality assurance standards for compliance, verification and validation.  The OPNFV Verified Program (OVP), by Linux Foundation Networking (LFN), overseen by the Compliance Verification Committee (CVC), will provide tracking and governance for RM/RA verification.
+
+For the purposes of this chapter, NFVI+VNF testing will be performed for **Verification**, **Validations**, ultimately leading to **Certifications** indicating a measured performance of the adherence to, and demonstrated proficiency with, all aspects of software delivery including but no limited to planning, logistics for communication, and testing of installed, or implmented, NFVI.
 
 <a name="1.1"></a>
-## 1.1 Overview
+## 1.1 Introduction
+
+**Chapter Purpose**<br>
+This chapter includes process flow, logistics, and requirements which must be satisfied to ensure Network Function Virtualisation Infrastructure (NFVI) meets the design, feature, and capability expectations of VNF developers promoting both the use and scalability of Software Defined Networking (SDN) capabilities.  Upstream projects will define features/capabilities, test scenarios, and test cases which will be used to augment OVP test harnesses for infrastructure verification purposes.  Existing processes, communication mediums, and related technologies will be utilized where feasible.  Ultimately, test results of certified NFVI+VNF will reduce the amount of time and cost it takes each operator to on-board and maintain vendor provided VNFs.
+
+**Objective**<br>
+Perform NFVI+VNF Verification and Validations using CNTT reference architecture, leveraging the existing OPNFV and CVC Intake and Validation Process to onboard and validate new test projects for NFVI compliance.  Upstream projects will define features/capabilities, test scenarios, and test cases to augment existing OVP test harnesses to be executed via the OVP Ecosystem.
 
-To be added once more content is developed.
+**Test Methodology**
+- Manifest Verifications (Termed Compliance by CVC) will ensure the NFVI is compliant, and delivered for testing, with hardware and software profile specifications defined by the Ref Model and Ref Architecture.
+- Empirical Validation with Reference Golden VNFs (Termed Validation by CVC) will ensure the NFVI runs with a set of VNF Families, or Classes, to minic production-like VNF connectivity, for the purposes of interopability checks.
```

Comment:
please change to verification from validation. thanks

Reply:
@mkr1481 , I believe Verification in this context is correct . The term Verification is utilized to perform a check against stated requirements.  i.e. did development provide a product which matches stated client component requirements.  Not that it performs correctly against FRs (Functional Requirements), but was it delivered per specs.  For Manifest verifications we're actually performing a check against the delivered Openstack components to ensure they match the RM/RA-1 requirements.  We perform Validations later with API/functional checks to make sure the APIs perform as expected (i.e. validate).

Reply:
and, @mkr1481 let me know if we can close this conversation.